Duplicati/Server/Program.cs

```diff
@@ -380,6 +380,8 @@ public static void RealMain(string[] args)
                 if (commandlineOptions.ContainsKey("webservice-password"))
                     Program.DataConnection.ApplicationSettings.SetWebserverPassword(commandlineOptions["webservice-password"]);
 
+                Program.DataConnection.ApplicationSettings.GenerateWebserverPasswordTrayIcon();

```

Comment:
Whole feature logic is:
1) When server is started, it generate random password for tray icon (separate from password for login form). Because user dont need to know password (it is only for confirmation, that tray know secret from database) it is not problem that is generated for all users (of course when user disable password at all, even tray password will be empty)
2) SERVER_PASSPHRASE_SALT is set by SetWebserverPassword. So when password is empty, salt is empty.
3) TrayIconPassword share SALT with FormPassword